### PR TITLE
update schedule

### PR DESCRIPTION
**Pull request by andrewsunglaekim**
Changed repo name from pbj to wdi, changed some things that reference it
Changes some verbage about feedback expectations
Updates Schedule

Also, my english and grammar suck, so fix anything you'd like or make it sound better.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
+#### Mon - 11 Jan
+Schedule 15 minute appt with instructor. Attendance will be taken at appt by squad leader.
+
+#### Tue(12 Jan) - Thur(14 Jan)
+Schedule 30 minute appt with instructor. Attendance will be taken at appt by squad leader.
+
+#### Friday
+|time| what's happening|
+|---|---|
+|9:00| breakfast? |
```
Comment (andrewsunglaekim) [inline readme.md]: can we do this @bmartinowich ?

File: readme.md [hunk 2]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
+|12:00| Pencil's Down! Lunch is provided! |
+|1:00 - 1:45| NaN, AwesomeAnimalHomies, FullSquid`<div>`elopers presents |
+|1:45 - 2:00| Break! |
+|2:00 - 2:45| Undefined, DrJammer, Hackasaurs presents |
+|2:45 - 3:00| Break! |
+|3:00 - 3:45| `<br>`ingBad, LearningToShred, Skwahd presents |
+|4:00 - 4:30| Final Squad Scrum |
+|4:30 - 5:00| End Course Feedback |
```
Comment (andrewsunglaekim) [inline readme.md:121]: not sure if we need 30 minutes for this, but put 30 for now

File: readme.md [hunk 3]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
-- Friday
-  - 12:00: Attendance will be taken. You and another student will pair to peer-review each others' code.
-  - 12:30 Lunch is provided!
-  - 3:00: Squads Ada, Bash, and C will present at the Science Fair.
-  - 3:30: Break
-  - 3:45: Squads Dart, Elixir, and Fortran will present at the Science Fair.
-  - 4:15: Break
-  - 4:30: The class will debrief as a whole.
-  - 5:30: The start of the rest of your lives! (And Happy Hour!)
+### Schedule - Dates and times
```
Comment (andrewsunglaekim) [inline readme.md:93]: Schedule does not include any mini lessons

File: readme.md [hunk 4]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
+|9:00| breakfast? |
+|12:00| Pencil's Down! Lunch is provided! |
+|1:00 - 1:45| NaN, AwesomeAnimalHomies, FullSquid`<div>`elopers presents |
+|1:45 - 2:00| Break! |
+|2:00 - 2:45| Undefined, DrJammer, Hackasaurs presents |
+|2:45 - 3:00| Break! |
+|3:00 - 3:45| `<br>`ingBad, LearningToShred, Skwahd presents |
+|4:00 - 4:30| Final Squad Scrum |
+|4:30 - 5:00| End Course Feedback |
+|5:00| The class will debrief as a whole.|
```
Comment (andrewsunglaekim) [inline readme.md:122]: we have happy hour... right @bmartinowich ?

File: readme.md (hunk 3 above)
Comment (adambray) [inline readme.md:93]: That's fine for now, but I'd propose doing one or two.

File: readme.md [hunk 5]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
-  - 3:30: Break
-  - 3:45: Squads Dart, Elixir, and Fortran will present at the Science Fair.
-  - 4:15: Break
-  - 4:30: The class will debrief as a whole.
-  - 5:30: The start of the rest of your lives! (And Happy Hour!)
+### Schedule - Dates and times
+#### Fri - 8 Jan
+|time| what's happening|
+|---|---|
+|9:00| Project 4 Kickoff. Pick whichever classroom you want! |
```
Comment (adambray) [inline readme.md:97]: Any reason to kickoff before Friday?

File: readme.md [hunk 6]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
+Schedule 15 minute appt with instructor. Attendance will be taken at appt by squad leader.
+
+#### Tue(12 Jan) - Thur(14 Jan)
+Schedule 30 minute appt with instructor. Attendance will be taken at appt by squad leader.
+
+#### Friday
+|time| what's happening|
+|---|---|
+|9:00| breakfast? |
+|12:00| Pencil's Down! Lunch is provided! |
```
Comment (adambray) [inline readme.md]: s/Pecil's/Pencils

File: readme.md [hunk 7]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
+|1:00 - 1:45| NaN, AwesomeAnimalHomies, FullSquid`<div>`elopers presents |
+|1:45 - 2:00| Break! |
+|2:00 - 2:45| Undefined, DrJammer, Hackasaurs presents |
```
Comment (adambray) [inline readme.md]: s/Undefined/Four Leaf Coders

File: readme.md (hunk 4 above)
Comment (adambray) [inline readme.md:122]: @bmartinowich just noticed there's a GA Open House that day? That gonna conflict?

File: readme.md [hunk 8]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
 
-Instructors will not be grading this project. We want you to use this final project as your magnum opus
-as you apply for jobs. 
+Instructors will not be grading this project. We want you to use this final project as your magnum opus as you apply for jobs.
 
-If you would like instructor feedback after your project has been completed, please **submit a link to your Github repository as a Git issue to the [`pbj-project4-gallery`](https://github.com/ga-dc/pbj-project4-gallery) repository**. If you do so **by 6:00pm on Friday**, we will do our best to have feedback returned to you by Monday morning. If you do so afterward, please just Slack an instructor from whom you'd like feedback to let them know.
+If you would like instructor feedback after your project has been completed, please **submit a link to your Github repository as a Git issue to the [`wdi-project4-gallery`](https://github.com/ga-dc/wdi-project4-gallery) repository**. If you do so **by 6:00pm on Friday**, we will do our best to have feedback returned to you some time the following week. If you do so afterward, please just Slack an instructor from whom you'd like feedback to let them know.
```
Comment (jshawl) [inline readme.md]: should have explicit deadline. Recommend 1 business day after instructor break

File: readme.md [hunk 9]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
 
-As always, instructors will be available for 1-1 appointments and support. Please **submit technical issues to the `pbj-project4` repository as Git issues**. To schedule an appointment, please Slack the relevant instructor.
+As always, instructors will be available for 1-1 appointments and support. Please **submit technical issues to the `project4` repository as Git issues**. To schedule an appointment, please Slack the relevant instructor.
```
Comment (jshawl) [inline readme.md]: are we just doin 1-1s on a per-request basis? im into that

File: readme.md [hunk 10]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
+|---|---|
+|9:00| Project 4 Kickoff. Pick whichever classroom you want! |
+|9:00 - 11:00| Project Brainstorm - user stories, some initial sketches/wireframing, possible ERD's|
+|11:30| Squad Scrum - Bounce stories off squad and instructor
+|11:00 - 4:00| Work on projects |
+|4:00| Feed BACK(robin's activity)|
+|5:00| Squad Scrum - Discuss project 1 |
+
+#### Mon - 11 Jan
+Schedule 15 minute appt with instructor. Attendance will be taken at appt by squad leader.
```
Comment (jsm13) [inline readme.md]: Phrasing sounds like the scheduling of the meeting happens Monday. I assume the intent is that the 15 minute appt (which was scheduled Friday or over the weekend) will occur on Monday.

File: readme.md (hunk 6 above)
Comment (mattscilipoti) [inline readme.md]: s/Pecil's/Pencil's

File: readme.md (hunk 10 above)
Comment (ebirving) [inline readme.md]: +1

File: readme.md (hunk 6 above)
Comment (ebirving) [inline readme.md]: It's definitely Pencils

File: readme.md [hunk 11]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
-  - 12:00: Attendance will be taken. You and another student will pair to peer-review each others' code.
-  - 12:30 Lunch is provided!
-  - 3:00: Squads Ada, Bash, and C will present at the Science Fair.
-  - 3:30: Break
-  - 3:45: Squads Dart, Elixir, and Fortran will present at the Science Fair.
-  - 4:15: Break
-  - 4:30: The class will debrief as a whole.
-  - 5:30: The start of the rest of your lives! (And Happy Hour!)
+### Schedule - Dates and times
+#### Fri - 8 Jan
```
Comment (ebirving) [inline readme.md]: Friday (8 Jan)

File: readme.md [hunk 12]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
+|9:00 - 11:00| Project Brainstorm - user stories, some initial sketches/wireframing, possible ERD's|
+|11:30| Squad Scrum - Bounce stories off squad and instructor
+|11:00 - 4:00| Work on projects |
+|4:00| Feed BACK(robin's activity)|
+|5:00| Squad Scrum - Discuss project 1 |
+
+#### Mon - 11 Jan
+Schedule 15 minute appt with instructor. Attendance will be taken at appt by squad leader.
+
+#### Tue(12 Jan) - Thur(14 Jan)
```
Comment (ebirving) [inline readme.md]: s/Tue/Tuesday
s/Thur/Thursday

File: readme.md [hunk 13]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
+|time| what's happening|
+|---|---|
+|9:00| Project 4 Kickoff. Pick whichever classroom you want! |
+|9:00 - 11:00| Project Brainstorm - user stories, some initial sketches/wireframing, possible ERD's|
+|11:30| Squad Scrum - Bounce stories off squad and instructor
+|11:00 - 4:00| Work on projects |
+|4:00| Feed BACK(robin's activity)|
+|5:00| Squad Scrum - Discuss project 1 |
+
+#### Mon - 11 Jan
```
Comment (ebirving) [inline readme.md]: Monday (11 Jan)

File: readme.md [hunk 14]
```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
+Schedule 15 minute appt with instructor. Attendance will be taken at appt by squad leader.
+
+#### Tue(12 Jan) - Thur(14 Jan)
+Schedule 30 minute appt with instructor. Attendance will be taken at appt by squad leader.
+
+#### Friday
```
Comment (ebirving) [inline readme.md]: (15 Jan)

File: readme.md (hunk 4 above)
Comment (ebirving) [inline readme.md:122]: @bmartinowich Just a heads up -- I remember some butting of heads between UX and Campus Experience over space while UXDI's final retro overlapped with setup for Hacker Hustle Designer, seemly due to lack of pre-crunchtime communication on both side.

File: readme.md (hunk 8 above)
Comment (ebirving) [inline readme.md]: +1

File: readme.md (hunk 2 above)
Comment (ebirving) [inline readme.md:121]: I like 30

File: readme.md (hunk 1 above)
Comment (ebirving) [inline readme.md]: If yes, Breakfast should be capitalized.

#englishmajor4lyfe

File: readme.md (hunk 9 above)
Comment (ebirving) [inline readme.md]: This makes it sounds like the 1-1's are optional and potentially unlimited(?), whereas the references in the schedule sound mandatory and limited to two.

Comment (nolds9) [inline readme.md]: Add a header of `how to ask for help` and include talking points about 30 minute and 15 minute blocks for one-on-ones where attendance will be taken

File: readme.md (hunk 10 above)
Comment (nolds9) [inline readme.md]: Include 30 minute block to schedule sometime between Tuesday-Thursday

File: asking_for_help.md [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+```
+I tried ___.
+I expected ___ to happen.
+___ happened instead.
+My repo link is ___ and my question is about lines ___.
+```
+
+[Just click this handy link to create an issue with these blanks inserted for you!](https://github.com/ga-dc/project2/issues/new?title=issue+with+____&body=I+tried+_____.+I+expected+_____+to+happen._____+happened+instead.%0DMy+repo+link+is+___+and+my+question+is+about+lines+___.)
+
+*Please close your issues when you've received a satisfactory answer*, and let us know what the solution was! We'll close issues that have gone more than 3 hours without any updates.
```
Comment (adambray) [inline asking_for_help.md]: 3 hours (during the daytime) without _a response_ from students.

File: asking_for_help.md [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+Issues should follow this format:
+```
+I tried ___.
+I expected ___ to happen.
+___ happened instead.
+My repo link is ___ and my question is about lines ___.
+```
+
+[Just click this handy link to create an issue with these blanks inserted for you!](https://github.com/ga-dc/project2/issues/new?title=issue+with+____&body=I+tried+_____.+I+expected+_____+to+happen._____+happened+instead.%0DMy+repo+link+is+___+and+my+question+is+about+lines+___.)
+
```
Comment (amaseda) [inline asking_for_help.md:15]: This link has `project2` in it.

File: asking_for_help.md [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+**Note:** Issues can only be reopened by instructors. Therefore, should your issue get closed, please create a *new* issue, and if it's applicable to your previous issue, just include a link to the previous issue.
+
+## Why?
+Many reasons! The biggest is that programmers often have the same question. This way, students with the same question can find their answer in one place.
+
+Additionally, having to articulate the question into words is very helpful in understanding the problem. Sometimes, simply articulating the question gives you the answer!
+
+## How?
+Below is a guide for the information that must be included when submitting an issue. (You can also use the handy [template](https://github.com/ga-dc/project2/issues/new?title=issue+with+____&body=I+tried+_____.+I+expected+_____+to+happen._____+happened+instead.%0DMy+repo+link+is+___+and+my+question+is+about+lines+___.) above.) It is expected that you will try to debug/solve an issue prior to submitting a request for help on Github.
+
```
Comment (amaseda) [inline asking_for_help.md]: `project2` in this one too.

File: readme.md [hunk 18]
```diff
@@ -76,33 +76,48 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
 
-Instructors will not be grading this project. We want you to use this final project as your magnum opus
-as you apply for jobs. 
+Instructors will not be grading this project. We want you to use this final project as your magnum opus as you apply for jobs.
 
-If you would like instructor feedback after your project has been completed, please **submit a link to your Github repository as a Git issue to the [`pbj-project4-gallery`](https://github.com/ga-dc/pbj-project4-gallery) repository**. If you do so **by 6:00pm on Friday**, we will do our best to have feedback returned to you by Monday morning. If you do so afterward, please just Slack an instructor from whom you'd like feedback to let them know.
+If you would like instructor feedback after your project has been completed, please **submit a link to your Github repository as a Git issue to the [`wdi-project4-gallery`](https://github.com/ga-dc/wdi-project4-gallery) repository**. If you do so **by 6:00pm on Friday**, we will do our best to have feedback returned to by **COB Friday of the following week (1/22)**. If you do so afterward, please just Slack an instructor from whom you'd like feedback to let them know.
```
Comment (adambray) [inline readme.md]: Maybe add in that requests made after the deadline will take longer to give feedback on.

File: asking_for_help.md (hunk 15 above)
Comment (RobertAKARobin) [inline asking_for_help.md]: Half the time I close stuff after like 30 minutes. Maybe change it to:

> We'll close issues that have gone for a while without any updates from you.

**Comment by adambray** [comment]
Other than my comments, I feel good with this!

File: readme.md [hunk 19]
```diff
@@ -76,33 +76,48 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
 
-As always, instructors will be available for 1-1 appointments and support. Please **submit technical issues to the `pbj-project4` repository as Git issues**. To schedule an appointment, please Slack the relevant instructor.
+### Support + Issues
+
+Instructors will meet with students during scheduled one-on-ones, which will be with student's squad leader. Please **submit technical issues to the `project4` repository as Git issues**. If you're feeling stuck, please review the [How to Ask For Help Doc](asking_for_help.md)
```
Comment (RobertAKARobin) [inline readme.md]: ...will be with _the_ student's squad leader.

File: asking_for_help.md (hunk 16 above)
Comment (nolds9) [inline asking_for_help.md]: where?

File: readme.md [hunk 20]
```diff
@@ -76,33 +76,48 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
...
-  - 5:30: The start of the rest of your lives! (And Happy Hour!)
+### Schedule - Dates and times
+#### Friday (8 Jan)
+|Time| What's Happening|
+|---|---|
+|9:00| Project 4 Kickoff. Pick whichever classroom you want! |
+|9:00 - 11:00| Project Brainstorm - user stories, some initial sketches/wireframing, possible ERD's|
+|11:30| Squad Scrum - Bounce stories off squad and instructor
+|11:00 - 4:00| Work on projects |
+|4:00| Feed BACK(robin's activity)|
```
Comment (RobertAKARobin) [inline readme.md]: Remove `(robin's activity)`

File: asking_for_help.md (hunk 16 above)
Comment (nolds9) [inline asking_for_help.md:15]: :+1:

File: asking_for_help.md (hunk 15 above)
Comment (nolds9) [inline asking_for_help.md]: :+1:

File: readme.md (hunk 18 above)
Comment (nolds9) [inline readme.md]: Instead of "If you do so afterward, please just Slack an instructor from whom you'd like feedback to let them know" ?

**Pull request merged**